src/DynamoPublish/Views/PublishView.xaml

```diff
@@ -163,5 +188,13 @@
                     Margin="0,10,0,0" />
         </Grid>
 
+        <wfi:WindowsFormsHost Grid.Column="1"
```

Comment:
WPF doesn't support direct .gif loading. I used this StackOverflow article: http://stackoverflow.com/questions/210922/how-do-i-get-an-animated-gif-to-work-in-wpf